src/modules/test/recipes/compliance/FileRegexMatch.json

```diff
@@ -36,7 +36,7 @@
     "ObjectType": "Reported",
     "ComponentName": "Compliance",
     "ObjectName": "auditTest",
-    "Payload": "PASS{ FileRegexMatch: All facts exist } == TRUE"
+    "Payload": "PASS{ FileRegexMatch: All 1 files matched the pattern } == TRUE"
```

Comment:
>files [](http://example.com/codeflow?start=44&length=5)

nit: when there is only one file, can we say "file" instead of "files"?

Reply:
While poor grammar, having consistency here helps with automated testing.